demos-europe/demosplan-core
# F remove flysystem dependency

As I was checking #560 to see if we can update, it became apparent that we implement flysystem but then don't actual use it. So instead of updating and keeping it in case we might actually start using it sometime in the future, we decided to remove it (at least for now). It might make a glorious comeback in the future, who knows?

I've removed all usages and mentions of the dependency and tested the result with uploading and opening files in planning categories and announcements.

### How to review/test
Check out the branch and find any place that interacts with files. It should still work the same as before.

### Linked PRs (optional)

- [Documentation-PR](https://github.com/demos-europe/demosplan-documentation/pull/15)
- Dependabot-PR #560
